e2e/tests/up/testdata/docker-features-http-headers/.devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Test hello",
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",
+  "features": {
+    "@SERVER_URL@/devcontainer-feature-hello.tgz": {}
```

Comment:
super small nitpick: We use `#{}` to escape in `pkg/devcontainer/feature/extend.go`. Since it's related to this would be great to keep the escaping consistent

Reply:
done